lib/workers/repository/update/pr/__fixtures__/prbody2

```diff
@@ -0,0 +1,29 @@
+This PR contains the following updates:
+
+| Package | Type | Update | Change |
+|---|---|---|---|
+| [six](http://sixlang.org) ([source](https://togithub.com/matthewrobb/six)) | dependencies | patch | [`0.0.6` -> `0.0.12`](https://renovatebot.com/diffs/npm/six/0.0.6/0.0.12) |
+
+---
+
+### Configuration
+
+📅 **Schedule**: At any time (no schedule defined).
+
+🚦 **Automerge**: Disabled by config. Please merge this manually once you are satisfied.
+
+♻ **Rebasing**: Whenever PR becomes conflicted, or you tick the rebase/retry checkbox.
+
+🔕 **Ignore**: Close this PR and you won't be reminded about this update again.
+
+---
+
+ - [ ] <!-- rebase-check -->If you want to rebase/retry this PR, click this checkbox.
+
+---
+
+This PR has been generated by [Renovate Bot](https://togithub.com/renovatebot/renovate).
+<!-- {
```

Comment:
```suggestion

<!-- renovate-pr-data {
```

I think we should add this text to be more sure about the matching.

Reply:
yes

Reply:
Also change it so that it ends with `\n-->\n` so that we don't have to worry about matching the field names specifically within. The contents of this JSON will evolve over time (including additions and renames)

Reply:
i parse the JSON as is, i take it as a group and parse it, i dont match any contents inside

Reply:
Yes but you should avoid needing `prCreationVer` and `prUpdateVer` in your regex

Reply:
Yes